package.json

```diff
@@ -43,7 +43,7 @@
     "start:backend:gp2": "yarn workspace @asap-hub/gp2-server start",
     "start:frontend:gp2": "yarn workspace @asap-hub/gp2-frontend start",
     "start:storybook": "yarn workspace @asap-hub/storybook start",
-    "test": "LANG=en_US TZ=UTC jest",
+    "test": "LANG=en_US TZ=UTC LOG_LEVEL=silent jest",
```

Comment:
if you do that it will override the LOG_LEVEL setting from the .env file which I think is often helpful to have. Unless the .env config takes precedence?

Reply:
It will override `.env`, and deliberately so. When I'm _running_ the app I want super-verbose runtime logs so I have `LOG_LEVEL=debug` set in my `.env`, but I don't want any of that in a test run. Switching the value in `.env` in and out depending on context isn't really a good option.

Reply:
I agree. This has been annoying me for awhile

Reply:
except i don't know how easy it is to then override if i wanted to debug why my test is failing?

Reply:
i dont think we should be overriding it npm scripts as i expect env files to be source of truth for env var.  Doesn't running the following solve your issue without modifying the npm scripts
```
LOG_LEVEL=silent yarn test
```